original/v1/s012-pyglet/handout/handout.tex

```diff
@@ -0,0 +1,87 @@
+% arara: lualatex
+% arara: lualatex
+% Typeset using lualatex
+
+\documentclass[a4paper,10pt]{article}
+\usepackage[utf8]{luainputenc}
+\usepackage[pdfborder={0 0 0}]{hyperref}
+
+\usepackage[czech]{babel}
+
+\usepackage{fouriernc}
+\usepackage{tgpagella}
+\usepackage{xcolor}
+\usepackage[activate=true]{microtype}
+\usepackage{amssymb}
+\usepackage{fontspec}
+\usepackage{titling}
+\usepackage{titlesec}
+
+\usepackage{calc}
+\usepackage{multicol}
+\usepackage{array}
+\usepackage{enumitem}
+\usepackage{tikz}
+\usepackage{siunitx}
+\usepackage{colortbl}
+\usepackage{everypage}
+\usepackage[a4paper,margin=2cm,top=1cm]{geometry}
+
+\setmainfont[Numbers=OldStyle]{TeX Gyre Pagella}
+
+\definecolor{plpink}{HTML}{FF3232}
+\definecolor{plhome}{HTML}{CCEEFF}
+\definecolor{silver}{HTML}{888888}
+
+\sisetup{
+    output-decimal-marker={,}
+}
+\pagestyle{empty}
+
+\parskip=1ex
+\parindent=0pt
+
+\setlist[enumerate,1]{start=0}
+
+\AddEverypageHook{
+    \begin{tikzpicture}[remember picture,overlay]
+        \node[opacity=0.1,below left,xshift=1cm,yshift=1cm] at (current page.north east) {\includegraphics[width=12cm]{../../images/pylady-pink}};
+        \node[below left,align=right,yshift=-1.5em] at (current page.north east) {\color{white} strana \thepage};
+        \node[below left,align=right] at (current page.north east) {\color{white}};
+    \end{tikzpicture}
+}
+
+\newcommand\answerspace{\\\rule[0cm]{0pt}{1cm}}
+
+\newcommand\True{\texttt{True}}
+\newcommand\False{\texttt{False}}
+
+\usetikzlibrary{turtle}
+\newcommand\turtle[1]{\tikz[>=stealth,x=0.25mm,y=0.25mm]\draw[turtle={home,rt=90,#1}];}
+\newcommand\gulp[1]{}
+
+\newcommand\startsection[1]{
+     \vspace{0.2ex}
+    \hrule
+    {\fontspec{Oxygen} \tiny
+     \vspace{-1ex}
+     \emph{#1}
+     \vspace{-1.5em}
+    }
+}
+
+\newfontfamily\headingfont[]{Bree Serif}
+\titleformat*{\section}{\LARGE\headingfont}
+
+\begin{document}
+
+\section*{Domácí projekty}
+
+\begin{enumerate}
+\item Uprav projekt z hodiny tak, aby obrázek létal po okně a nikdy jej neopustil tzn.
+      aby nikdy neuletěl zcela mimo viditelnou plochu.
+      Obrázek může opisovat nějaký tvar (kružnici, čtverec atp.), odrážet se od odkrajů
```

Comment:
Možná bych raději řekl, že se může pohybovat po nějakém neviditelném tvaru? Opisoval je správně, ale možná si pod tím ne každý představí, co se tím myslí? Nevím...

Reply:
Pohybovat se po neviditelném tvaru mi přijde ještě krkolomnější popis. Nechám to tak a ujistím se na hodině, že zadání každý rozumí.

Reply:
Jo, to určitě je krkolomnější. OK